### PR TITLE
FEM lab: add layout-options component

### PR DESCRIPTION
**Pull request by goplayoutside3**
Staging branch URL: https://pr-6454.pfe-preview.zooniverse.org

## Describe Your Changes

This PR adds a layout-options component to the fem lab. The checkbox to enable `use_natural_subject_dimensions` in workflow.configuration is going toward handling a layout in FEM where subjects are displayed with their natural dimensions (at maximum) rather than stretching to fill a volunteer's browser.

Here's a screenshot of the text alongside the checkbox, and tagging @seanmiller26 for review:
Screenshot 2023-03-02 at 12 59 21 PM

## To Review 

On this PR's deployed branch: https://pr-6454.pfe-preview.zooniverse.org/lab/19645/workflows/22532?femLab=true&env=production

If reviewing locally, here's a link to a project's femLab, and be sure to use `HOST=local.zooniverse.org npm start`
https://local.zooniverse.org:3735/lab/19645/workflows/22532?femLab=true&env=production

By checking and unchecking the box, you can inspect `workflow.configuration.use_natural_subject_dimensions` in this FEM test project's network tab: https://frontend.preview.zooniverse.org/projects/goplayoutside3/fem-flipbook-viewer/classify/workflow/22532

### CODE REVIEW

File: app/pages/lab-fem/components/layout-options.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+export default function LayoutOptions({ workflow = null }) {
+  const handleNaturalSubjectDimesions = (e) => {
```
Comment (mcbouslog) [inline app/pages/lab-fem/components/layout-options.jsx:4]:
```suggestion
  const handleNaturalSubjectDimensions = (e) => {
```

File: app/pages/lab-fem/components/layout-options.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        By default, subject images stretch to fit the width of the volunteer's
+        browser. Check this option if you want the subject's maximum dimensions
+        to be the image's natural width and natural height.{" "}
+      </small>
+      <br />
+      <label>
+        <input
+          type="checkbox"
+          checked={useNaturalSubjectDimensions}
+          onChange={handleNaturalSubjectDimesions}
```
Comment (mcbouslog) [inline app/pages/lab-fem/components/layout-options.jsx:26]:
```suggestion
          onChange={handleNaturalSubjectDimensions}
```

File: app/pages/lab-fem/components/layout-options.jsx (hunk 1 above)
Comment (mcbouslog) [reply app/pages/lab-fem/components/layout-options.jsx:4]: In [fem-multi-image-subject-options-editor](https://github.com/zooniverse/Panoptes-Front-End/blob/c532227e78f788f6674480779cf8e95a53caf38d/app/pages/lab-fem/components/fem-multi-image-subject-options-editor.jsx#L18-L23) the handler is a regular function, maybe this should be as well for consistency, or if going to keep as arrow function, don't need `return`.

Optional suggestion is to change this to a regular function.

**Review by mcbouslog — APPROVED** [review]
Looks good!

A few comments to be addressed before merging, but none that I think would need additional review.

**Review by seanmiller26 — APPROVED** [review]

**Comment by goplayoutside3** [comment]
> Can we change the wording to say 'original dimensions' instead of 'natural'? Otherwise looks good!

Yes not a problem!  Thank you both I'll make the changes before merging.

**Pull request merged**